site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -11,7 +11,7 @@ A list of Connector policies. Users will be authenticated against Connectors in
 The identifier of the Connector to which this policy refers.
 
 [field]#{base_field_name}.connectorPolicies``[x]``.domains# [type]#[String]# [since]#Available since 1.18.0#::
-An list of email domains to which this connector should apply. 
+An list of email domains to which this connector should apply.
```

Comment:
I see `configured` in the json examples, but not documented here. Do we need to document that?